### PR TITLE
Add summary view, table mode, & evolution view functionality.

### PR DESCRIPTION
**Pull request by ecomerford**
@chutten in this PR, I have:

1. added more functionality to the summary view, including a `getChange()` function that calculates the % change, and conditional rendering that displays the change statement and arrow properly. 

2. updated the distribution view with a button group that toggles between the graph and the table, rather than a nav bar (a bit more complicated but I think it looks a lot better).

3. added functionality to the table mode, which now pulls in data and maps it into the body of the table.

4. added a line graph to the evolution view, and

5. given the whole thing a bit of styling (AKA a font).

The graphs still need a bit of work, and I'm having parsing issues when I try to fetch the data files from our GH pages, so I've just added a short bit of data for now.

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -12,6 +12,7 @@
     <link rel="shortcut icon" href="%PUBLIC_URL%/favicon.ico">
     <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous">
     <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.0.13/css/all.css" integrity="sha384-DNOHZ68U8hZfKXOrtjWvjxusGo9WQnrNx2sqG0tfsghAvtVlRW3tvkXWZh58N9jp" crossorigin="anonymous">
+    <link href="https://fonts.googleapis.com/css?family=Roboto+Mono" rel="stylesheet">
```
Comment (chutten) [inline public/index.html:15]: For headers and highlighted ares we could take a look at the Zilla Slab font mozilla had made for its rebranding last year: https://fonts.google.com/specimen/Zilla+Slab

File: src/Components/DistributionView.js [hunk 2]
```diff
@@ -1,54 +1,63 @@
...
+  handleChange = (event) => {
+    this.setState({mode : event,})
+  }
+
   render() {
     return (
       <Grid  className="dist-view" fluid>
         <Row>
-          <Col>
+          <Col >
```
Comment (chutten) [inline src/Components/DistributionView.js]: unnecessary whitespace

File: src/Components/DistributionView.js [hunk 3]
```diff
@@ -1,54 +1,63 @@
...
+            x_label={this.props.activeMetric}
+            y_label="Proportion of Users"
+            chart_type="histogram"
+            width={600}
+            height={250}
+            show_rollover_text={true}
+            y_accessor="proportion"
+            x_accessor="start"
+          />
+          }
```
Comment (chutten) [inline src/Components/DistributionView.js]: Not sure whether the style is to have the closing brace at the same indent level as the opening one or not, but it means the `TableMode` is indented one level deeper.

File: src/Components/ViewSelector.js [hunk 4]
```diff
@@ -8,41 +8,50 @@ import {EvolutionView} from "./EvolutionView.js";
...
-      <Tabs
-        defaultActiveKey={"sum"}
-        id="view-tabs"
-      >
-        <Tab
-          eventKey={"sum"}
-          title={<span><i className="fas fa-chart-pie"></i> Summary</span>}
+      <div>
+        <Tabs
+          defaultActiveKey={"sum"}
```
Comment (chutten) [inline src/Components/ViewSelector.js]: "sum" has its own meaning besides being a short-form for "summary" so if you'll forgive the pedantry I'd prefer "summary" as the eventKey for the summary Tab.

File: public/index.html (hunk 1 above)
Comment (ecomerford) [reply public/index.html:15]: Good call! I'll update this for my next PR 😃

File: src/Components/ViewSelector.js (hunk 4 above)
Comment (ecomerford) [reply src/Components/ViewSelector.js]: Makes sense; I went ahead and updated them all for consistency.

**Review by chutten — APPROVED** [review]
